extensions/wikia/Listusers/SpecialListusers_body.php

```diff
@@ -40,7 +41,7 @@ function  __construct() {
 	 * show form
 	 */
 	public function execute( $subpage ) {
-		global $wgOut, $wgRequest, $wgCityId;
+		global $wgOut, $wgCityId;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app